tests/virt/thinp_test.py

```diff
@@ -24,7 +24,7 @@
 from vdsm.virt import thinp
```

Comment:
"What we did test is _monitored_volumes()" probably better to use "In the past we tested _monitored_volumes()"